MVCDemos/Views/Form/GroupedFields.cshtml

```diff
@@ -41,6 +45,7 @@
 
             items.AddGroup()
                 .Caption("Contact Information")
+                .CaptionTemplate(new JS("function(data) { return groupCaptionTemplate('tel',data); }"))
```

Comment:
As an example of how you can declare captionTemplate, use the 'Form/CustomizeItem' demo